pkg/maps/ctmap/ctmap.go

```diff
@@ -187,6 +187,14 @@ type GCFilter struct {
 // EmitCTEntryCBFunc is the type used for the EmitCTEntryCB callback in GCFilter
 type EmitCTEntryCBFunc func(srcIP, dstIP netip.Addr, srcPort, dstPort uint16, nextHdr, flags uint8, entry *CtEntry)
 
+// TODO: Remove this hack once ctmap is migrated to a cell.
```

Comment:
Please create and reference a GH issue here - TODOs in code without a reference to an issue are easily lost.